ui/app/common/attributeTypes/views/attributeInformation.html

```diff
@@ -1,84 +1,117 @@
-<div class="field-attribute" ng-if = "attribute.name != 'NICK_NAME'">
+<div class="field-attribute" ng-if="attribute.name != 'NICK_NAME'">
     <label for="{{::attribute.name}}">{{::attribute.description}} <span ng-show="isForm">:</span>
 
         <span class="asterick" ng-show="{{::attribute.required}}">*</span></label>
 </div>
 <div class="field-value" data-ng-switch on="::attribute.format">
-    <div data-ng-switch-when="java.lang.String">
+    <div data-ng-switch-when="java.lang.String" ng-if="attribute.name != 'occupation'">
```

Comment:
Can you change the attribute occupation to something like `PATIENT_OCCUPATION` ? Like what is done for `NICK_NAME` . We need to follow one naming convention while naming person attributes